### ISSUE TITLE
Can you provide the log of training?

### ISSUE THREAD

**Issue by yichen928** [PRBonn/TARL#9]
Hi, thanks for open-sourcing this great project! Can you provide the training log (including the loss curve) of the pretraining? It would be very helpful. Thank you very much!

**Comment by nuneslu** [comment]
Sure, here is the tensorboard [log](https://github.com/PRBonn/TARL/files/12511302/events.out.tfevents.1658304695.ipb19.1375072.zip) file and together some of the plots:

![image](https://github.com/PRBonn/TARL/assets/6524770/4d9df090-f9a8-449f-8baf-418c6893060d)
![image](https://github.com/PRBonn/TARL/assets/6524770/056b0135-9a61-49dc-be83-702c017ca12a)
![image](https://github.com/PRBonn/TARL/assets/6524770/8143e969-478d-4dd9-8ee4-b8db969dce84)

**Comment by yichen928** [comment]
Thanks very much for your reply!
